### ISSUE TITLE
[JTable] TableHeader does not scroll along with the body when model set after JTable initialised

### ISSUE THREAD

**Issue by tychobrailleur** [weisJ/darklaf#163]
**Describe the bug**
Following up on https://github.com/akasolace/HO/issues/85#issuecomment-619389962, I managed to create a small reproducer of the issue:

![scroll](https://user-images.githubusercontent.com/114285/80955259-e44dda00-8df6-11ea-8c6f-4fd05a6fae7b.gif)

This seems to happen when setting the model after instantiating the table, i.e.:

```
            JTable aTable = new JTable();
            aTable.setModel(new DarkTableModel(10,10));
```

(the full example can be found here: https://gist.github.com/tychobrailleur/621e82d08d2da2b5b3074da84a34b433)

**To Reproduce**
Cf. https://gist.github.com/tychobrailleur/621e82d08d2da2b5b3074da84a34b433

**Screenshots**
See above.

**Additional Information:**
 - OS: macOS
 - OS Version: 10.14.6
- Darklaf Version: 2.1.1

**Additional context**
N/A

**Comment by weisJ** [comment]
This doesn't happen for me.  Do you have access to a windows machine to check if you can reproduce the issue there? This might be related to macOS only. (Although I can't think of a reason why).

**Comment by akasolace** [comment]
@weisJ  I can confirm the issue under windows ...

**Comment by weisJ** [comment]
Which version of Java is used?

**Comment by akasolace** [comment]
java 8

**Comment by akasolace** [comment]
@weisJ  @tychobrailleur  just tested with openJDK 11 and the bug not here anymore ....

**Comment by tychobrailleur** [comment]
> @weisJ @tychobrailleur just tested with openJDK 11 and the bug not here anymore ....

Ah mystery solved!

**Comment by weisJ** [comment]
The fix is now available with `2.2.0`
